src/frontend/js/modules/auth/oidc/strategy.js

```diff
@@ -0,0 +1,54 @@
+import { UserManager } from 'oidc-client';
+
+class OIDCStrategy {
+  constructor(options) {
+    this.oidcUserManager = new UserManager(options);
+    this.oidcUserManager.getUser().then(user => {
```

Comment:
Should have error handle here?

Reply:
I am going to move this block, just because we are not sure that it gets user when we need it